lively.morphic/svg-morph/svg-morph.js

```diff
@@ -162,6 +164,63 @@ export class SVGMorph extends Morph {
 
   get isSVGMorph () { return true; }
 
+  getControlPoints () {
+    this.removeAllControlPoints();
+
+    const targetPath = SVG(this.target).array();
+    for (let i = 0; i < targetPath.length; i++) {
+      let element = targetPath[i];
+      console.log(element);
+      switch (element[0]) {
+        case 'M':
+          let mPoint = this.createControlPointAt(i, element[1], element[2], 'red');
+
+          this.svgPath.parentNode.appendChild(mPoint);
+
+          break;
+        case 'C':
+          let cPoint = this.createControlPointAt(i, element[5], element[6], 'yellow');
+
+          this.svgPath.parentNode.appendChild(cPoint);
+          break;
+        case 'L':
+          let lPoint = this.createControlPointAt(i, element[1], element[2], 'yellow');
+
+          this.svgPath.parentNode.appendChild(lPoint);
+          break;
+        case 'Z':
+          break;
+        default:
+          let defaultPoint = this.createControlPointAt(i, element[element.length - 2], element[element.length - 1], 'yellow');
+
+          this.svgPath.parentNode.appendChild(defaultPoint);
+      }
+      let lastElement = element;
+    }
+  }
+
+  createControlPointAt (id, x, y, color) {
+    let point = document.createElement('div');
+    point.className = 'control-point';
+    point.id = 'control-point-' + id;
+    point.style.width = '10px';
+    point.style.height = '10px';
+    point.style.background = color;
+    point.style.position = 'absolute';
+    point.style.left = y + 'px';
+    point.style.top = x + 'px';
+    // point.style.background-radius = "25px"
+
+    return point;
+  }
+
+  removeAllControlPoints () {
```

Comment:
cant we use something like: `$('control-point.my-path-selection').remove();`

Reply:
where do we get the jQuery from?

Reply:
this way it also works even though we might have only 1 or no controlPoints and we don't get a collection